data/hai/hai reveal 5 philosophers.txt

```diff
@@ -795,20 +795,114 @@ mission "Hai Reveal [B11] Moving Together"
 
 
 # Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
-ship "Shield Beetle" "Shield Beetle (Sunbeam Korath Jump)"
+ship "Emperor Beetle"
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		cost 35100000
+		shields 26700
+		hull 19600
+		"required crew" 83
+		bunks 91
+		mass 870
+		drag 9.8
+		"outfit space" 891
+		"cargo space" 27
+		"weapon capacity" 352
+		"engine capacity" 139
+		# Expensive features inherited from Ladybug-era technology, with some fuel (and improved heat dissipation):
+		"heat dissipation" 0.7
+		"tactical scan power" 30
+		ramscoop .5
+		"fuel capacity" 600
+		weapon
+			"shield damage" 5000
+			"hull damage" 2500
+			# High hit force and wide radius for a spectacle if the leader dies.
+			# This should be adjusted downward if the ship is reused more widely.
+			"blast radius" 800
+			"hit force" 8000
 	outfits
-		"Large Heat Shunt" 2
+		"Tripulse Shredder" 2
+		"Ionic Blaster Prototype" 4
+		"Pulse Turret" 4
+		"Warder Anti-Missile" 2
+		
+		"Large Heat Shunt"
+		"Fuel Processor"
 		"Jump Drive"
-		"Systems Core (Large)"
 		"Quantum Keystone"
-		"Outfits Expansion" 3
-		"Boulder Reactor" 3
-		`"Bufaer" Atomic Steering`
-		Sunbeam 6
-		"Korath Warder"
-		"Small Heat Shunt"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Wanderer Ramscoop"
+		
+		"White Sun Reactor"
+		"Large Biochemical Cell"
+		
+		"Afterburner (Comet Class)"
+		"Steering (Planetary Class)"
 		"Thruster (Planetary Class)"
-		"Fuel Processor" 2
+		
+		"Pulse Rifle" 65
+		"Fragmentation Grenades" 65
+		"Thermal Repeater Rifle" 25
+	engine -43 131
+		zoom 0.85
+	engine 43 131
+		zoom 0.85
+	engine 0 126.5
+		zoom 0.5
+	"reverse engine" -61 -18.5
+		angle -35
+	"reverse engine" 62 -18.5
+		angle 35
+	"steering engine" -61 -18.5
+		zoom 0.6
+		angle 145
+		left
+	"steering engine" 62 -18.5
+		zoom 0.6
+		angle 215
+		right
+	"steering engine" -73 46
+		zoom 0.5
+		angle 80
+		left
+	"steering engine" 73 46
+		zoom 0.5
+		angle -80
+		right
+	"steering engine" -43 131
+		zoom 0.45
+		right
+	"steering engine" 43 131
+		zoom 0.45
+		left
+	gun -42.5 -53.5 "Tripulse Shredder"
+	gun 43.5 -54.5 "Tripulse Shredder"
+	gun -23 -102.5
+	gun 24.5 -102.5
+	gun 12.5 -128
```

Comment:
```suggestion
	gun 12.5 -128
```

Though I think perhaps the Tripulses should be moved be moved into these center ports? Unless intent is to have them matched to the port designs??

Reply:
yeah that was the intent